addons/mail/static/src/components/messaging_menu/messaging_menu.scss

```diff
@@ -110,6 +110,7 @@
 
 .o_MessagingMenu_dropdownMenu.o-mobile {
     border: 0;
+    z-index: $zindex-tooltip + 10;
```

Comment:
Hello, thank you for the fix. Please add a comment to explain why, because we will never remember otherwise.

Also I would suggest to place the fix above in the file, closer to the other (non-mobile) `z-index` value.

And just to make sure, how comes the leafset doesn't have an issue in desktop? It has a different `z-index` value itself?

Reply:
Hi Seb,

It's doesn't affect desktop because a z-index value is already given see:
https://github.com/odoo/odoo/blob/5c52831b9dde0f159da04097e0a0ecaa201025f3/addons/mail/static/src/components/messaging_menu/messaging_menu.scss#L33
On mobile, it's displayed like fullscreen => I think there was no z-index applied before because of that. 

But unlucky for us leafset internally applied some z-index like you can see here: https://github.com/odoo/enterprise/blob/17121abb9eabc5d7ab5ce9e870abb1081112ff06/web_map/static/lib/leaflet/leaflet.css#L125L129
![image](https://user-images.githubusercontent.com/42938544/154481726-4c5696bf-8fb8-4bc0-8d34-f5171a9e96e6.png)

=> I have applied the same z-index value used here: https://github.com/odoo/enterprise/blob/17121abb9eabc5d7ab5ce9e870abb1081112ff06/mail_enterprise/static/src/scss/mail_enterprise_mobile.scss#L29

But I can apply the same z-index used on desktop if you prefer it's also fixes the visual problem

Reply:
The less code the better, if having the same z-index as desktop fixes the issue too, then I would do that instead. So just move the z-index out of the desktop condition?

Reply:
But it might not work when opening a chat window (either version actually), make sure that when opening the messaging menu in mobile, and then opening a channel from there, the channel is visible and not the menu.

Reply:
Currently, it's working well inside a chat window because the o_ChatWindowManager class has a z-index at the same value used by leafset. 

I will move the z-index value outside the desktop-only scope, it seems good to me.